cmd/lakectl/cmd/common_helpers.go

```diff
@@ -267,3 +271,16 @@ func MustParsePathURI(name, s string) *uri.URI {
 	}
 	return u
 }
+
+func IsValidAccessKeyID(accessKeyID string) bool {
+	return accessKeyRegexp.MatchString(accessKeyID)
+}
+
+func IsValidSecretAccessKey(secretAccessKey string) bool {
+	return IsBase64(secretAccessKey) && len(secretAccessKey) == 40
```

Comment:
Comment thread gone because the line changed...



Cool. But then please return a string explaining to the user what is suspicious about the URL: is it impossible to parse (which is definitely wrong), or does it have a weird suffix (which is probably wrong). The user can and should take different steps in each case.

Reply:
We might still want to do this :-)